### DISCUSSION TITLE
ChromiCraft Web Page visual issue

### DISCUSSION THREAD

**Discussion by Saikur0n** [chromiecraft/chromiecraft#3518] [General]
Hello. 
I've noticed a scroll bar bug in google chrome on your web page.
If you proceed to the 'Server Status' section in google chrome you will notice that there is two scroll bars instead of one. One for the list of players and second one for the page itself. In latest firefox browser there is only one main scroll bar. It looks weird to have two scroll bars in chrome in my opinion.
Somebody can notice about this to devs of the project?

**Comment by nicoaravena** [comment]
Firefox has improved their scrollbars as you can see in their [Firefox release notes](https://www.mozilla.org/en-US/firefox/100.0/releasenotes/), but if you see in detail, u will notice 2 scrollbars anyways. That is because most of the tools are iframes, that loads other azerothcore services, like the [server-status](https://github.com/azerothcore/server-status).

Edit: probably in higher resolutions u will not see 2 scrollbars.

**Reply by Saikur0n** [reply]
Oh, thank you for explanation. I thought this is just browser compatibility issue. :)
